chain/client/src/chunk_inclusion_tracker.rs

```diff
@@ -57,42 +99,47 @@ impl ChunkInclusionTracker {
     }
 
     /// Function to return the chunks that are ready to be included in a block.
-    /// We filter out the chunks that are produced by banned chunk producers.
-    /// Return type contains some extra information needed for debug logs and metrics.
-    ///     HashMap from [shard_id] to (chunk_header, chunk received_time, chunk_producer account_id)
+    /// We filter out the chunks that are produced by banned chunk producers or have insufficient
+    /// chunk validator endorsements.
+    /// Return HashMap from [shard_id] -> chunk_hash
+    /// We can later use the chunk_hash to fetch chunk_header, chunk_endorsements, chunk_producer_and_received_time
     pub fn get_chunk_headers_ready_for_inclusion(
-        &self,
+        &mut self,
```

Comment:
As I said in other comment, I don't expect `get` to mutate state.

Reply:
Done